apps/hashdotai/blog/039_self-building-database.mdx

```diff
@@ -0,0 +1,128 @@
+
+Over time, this database would check, validate and enrich your information, ensuring its freshness and accuracy, rather than letting it become stale (as happens to data in every other database, ever). Raw energy, via compute, is converted into beautiful negentropy.
+
+When it comes to sharing your information with other people, businesses, or service-providers, the database lets you do so with just one click. And when you want to remove their access, it's equally simple.
+
+Information from existing external applications and databases can be trivially two-way synced with the database, and the synced data represented as semantic entities for easy management and use.
+
+# Example use cases
+
+This kind of a database should be able to help with all of the below use cases.
+
+## Inferring novel statistics
+
+For many kinds of commercially-valuable information, providers of aggregated data and statistics already exist. For example, in the case of the financial markets, _LSEG Data & Analytics_ or S&P's _CapIQ_ provide many forms of standardized data.
+
+But for more idiosyncratic data, information may not be available "off-the-shelf" or in a pre-prepared form. In those cases, organizations often spend significant time compiling this data, leading to delays in decision-making, and not inconsiderable expense (even when research is offshored, as is commonly the case).
+
+Calculating metrics and inferring novel statistics may involve:
+
+- Collating hard-to-gather information from many different websites or sources
+- Talking to or requesting inputs from a wide array of actors, including both subjects and observers
+- Synthesizing gathered information into a clean, consistently structured form
+- Performing calculations or analysis using the gathered data
+
+Producing these statistics, that are not readily available to others, can support better informed decision-making, and provide alpha to traders, as well.
+
+## Enriching datasets
+
+Many organizations have "analysts" or "associates" whose roles involve large-scale data collection, research and analysis of individuals, companies, opportunities, and ideas.
+
+For example, a venture capitalist may want to conduct research on all of the authors of a particular academic paper. Within frontier AI research groups, for example, it is not uncommon for certain papers to have 10-20 (or more!) authors. Being able to take only their names and organizational affiliations, and automatically identify their LinkedIn profile, Google Scholar profile, Twitter profile, GitHub profile, current title, previous employers, and so on would save hours of manually scouring the web, freeing analysts up to conduct more meaningful due diligence.
+
+## Monitoring and tracking
+
+Automated monitoring and updating of information is one of our self-building database's core competencies.
```

Comment:
When I first read this, it seemed like a sudden switch to talking about 'our' as in 'our product'... but we return to the "imagined" theme later, so I assume it's meant to be read more like this
```suggestion
Automated monitoring and updating of information would be one of this self-building database's core competencies.
```